### PR TITLE
feat: add stepped debugger panel to sidebar

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

Debugging recordings can be difficult when you can't step through snapshot by snapshot

## Changes

Adds "debug snapshots" feature to allow you to walk through a recording on a per snapshot basis

### CODE REVIEW

File: frontend/src/scenes/session-recordings/player/sidebar/PlayerSidebarDebuggerTab.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    const { debugSnapshots } = useValues(sessionRecordingPlayerLogic)
+    const { setPause, seekToTimestamp } = useActions(sessionRecordingPlayerLogic)
+
+    useEffect(() => {
+        setPause()
+    })
+
+    if (!debugSnapshots) {
+        return <div>Not ready yet</div>
+    }
```
Comment (pauldambra) [inline frontend/src/scenes/session-recordings/player/sidebar/PlayerSidebarDebuggerTab.tsx:20]: probably less confusing to have a loader - only a debug view so no biggie

File: frontend/src/scenes/session-recordings/player/sidebar/PlayerSidebarDebuggerTab.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            <div className="border-y">
+                <LemonCollapse
+                    size="xsmall"
+                    multiple
+                    embedded
+                    panels={[
+                        {
+                            key: 'previous',
+                            header: 'Previous snapshot',
+                            content: previous && <JSONViewer src={previous} />,
```
Comment (pauldambra) [inline frontend/src/scenes/session-recordings/player/sidebar/PlayerSidebarDebuggerTab.tsx:62]: Ideally we'd handle absence here so it's clearer than a blank row

**Review by pauldambra — APPROVED** [review]
this is great i can't wait to use it in anger

**Comment by daibhin** [comment]
@pauldambra fixed!
fixed

**Pull request merged**